Admin/Client/src/components/navigation/requestBarItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import Request from '../../models/request/Request';
+import HttpMethodBadges from '../shared/HttpMethodBadges';
+
+type Props = {
+  request: Request;
+  isSelected: Boolean;
+  onRequestSeleceted: Function;
```

Comment:
I would put this type more specific - it's a function that gets request
` (request: Request) => void`

Reply:
Will change :)